### PR TITLE
Fix "No loader found for resource: res://" spam when NativeScript script_class_name is not empty.

### LINKED ISSUE #26275: "No loader found for resource: res://" when NativeScript has Script Class name
**Godot version:**
3.1 beta 5

**OS/device including version:**
OSX

**Issue description:**
I get spammed with the error as soon as I click "Add Node" and while I'm typing in the "Add Node" search.

I used lldb with VSCode to figure out where the `No loader found for resource: res://` was coming from (it'd be cool to get some kind of hint as to "who" is trying to load this resource).

I found out that it's somehow coming from the NativeScript classes I have. As in, there's one `No loader found ...`  for each `.gdn` I'm using. All `.gdn`s have a non-empty class name, script name, valid library path, valid resource path to itself and a resource name under the inspector.

Here's an example of one of my `.gdn`s:
```
[gd_resource type="NativeScript" load_steps=2 format=2]

[ext_resource path="res://bin/lutlib.gdnlib" type="GDNativeLibrary" id=1]

[resource]

resource_name = "CharacterAStar"
class_name = "CharacterAStar"
library = ExtResource( 1 )
script_class_name = "CharacterAStar"
```

Here's how the project.godot looks like for defining this global class name:
```
{
"base": "Reference",
"class": "CharacterAStar",
"language": "NativeScript",
"path": "res://src/lut/native/scripts/character_astar.gdns"
}
```

The spamming seems to really slow down the "Add Node" window. 

**Steps to reproduce:**
1. Create project with a NativeScript
2. Give the NativeScript a Script Class name
3. Right click any node in the Scene tab, and click Add Node
4. Look at Output

This issue is similar to https://github.com/godotengine/godot/issues/22377

### PR DESCRIPTION
**Pull request by ericrybick**
fixes #26275

### CODE REVIEW

File: modules/gdnative/nativescript/nativescript.cpp [hunk 1]
```diff
@@ -1599,18 +1599,20 @@ bool NativeScriptLanguage::handles_global_class_type(const String &p_type) const
...
 		if (r_icon_path)
-			*r_icon_path = script->get_script_class_icon_path();
-		return script->get_script_class_name();
-	}
-	if (r_base_type)
-		*r_base_type = String();
-	if (r_icon_path)
-		*r_icon_path = String();
+			*r_icon_path = String();
+		}
```
Comment (bojidar-bg) [inline modules/gdnative/nativescript/nativescript.cpp]: Wrong indentation here.

File: modules/gdnative/nativescript/nativescript.cpp [hunk 2]
```diff
@@ -1599,18 +1599,20 @@ bool NativeScriptLanguage::handles_global_class_type(const String &p_type) const
 }
 
 String NativeScriptLanguage::get_global_class_name(const String &p_path, String *r_base_type, String *r_icon_path) const {
-	Ref<NativeScript> script = ResourceLoader::load(p_path, "NativeScript");
-	if (script.is_valid()) {
+	if (!String(p_path).empty()) {
```
Comment (karroffel) [inline modules/gdnative/nativescript/nativescript.cpp]: does `!p_path.empty()` not work here?

Comment (akien-mga) [reply modules/gdnative/nativescript/nativescript.cpp]: And you can avoid indenting everything by doing instead:
```cpp
if (p_path.empty()) {
	String();
}
```

Comment (akien-mga) [reply modules/gdnative/nativescript/nativescript.cpp]: No nevermind, bad idea :)

Comment (ericrybick) [reply modules/gdnative/nativescript/nativescript.cpp]: you guys are fast 😄 , done.

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**